### DISCUSSION TITLE
Is there a plan to KMM target for iOS ?

### DISCUSSION THREAD

**Discussion by ivanpagac** [walt-id/.github#70] [Help] [answered]
Hello there, 
I saw you are developing KMM libraries. Do you plan to support iOS target in near future? 

Thanks in advance,
Ivan

**Comment by taminobaumann** [comment, marked as answer]
Hey @ivanpagac, 

thanks for reaching out! 

We plan to support IOS in the future, but we don't have an exact timeline for it yet. 

Please refer to our [roadmap](https://walt-id.notion.site/fcde1687baab42378b3047d4a22eeaca?v=1140dd17c17b4726a70cc1465d20866d) for updates.


Best
Tamino
